### PR TITLE
MODULES-4324 - README Updates for Translation.

### PR DESCRIPTION
**Pull request by pmcmaw**
This includes updates to the README to prepare for being sent away for
Translation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -666,100 +795,161 @@ Specifies the addresses the server accepts connections to. Valid values:
...
-Whether to manage the pg_hba.conf. If set to true, Puppet overwrites this file. If set to false, Puppet does not modify the file. Valid values: true, false. Default
+Whether to manage the `pg_hba.conf`.
+
+If set to `true`, Puppet overwrites this file.
+
+If set to `false`, Puppet does not modify the file.
+
+Valid values: `true`, `false`.
+
+Default value: TODO
```
Comment (jbondpdx) [inline README.md]: Defaults to the value in globals set in `postgresql::globals::manage_pg_hba_conf` (which defaults to `true`).

File: README.md [hunk 2]
```diff
@@ -666,100 +795,161 @@ Specifies the addresses the server accepts connections to. Valid values:
...
+Specifies the port for the PostgreSQL server to listen on. **Note:** The same port number is used for all IP addresses the server listens on. Also, for Red Hat systems and early Debian systems, changing the port causes the server to come to a full stop before being able to make the change. 
+
+Default value: 5432. Meaning the Postgres server listens on TCP port 5432.
 
 ##### `postgres_password`
 
-Sets the password for the `postgres` user to your specified value. Default: undef, meaning the superuser account in the Postgres database is a user called `postgres` and this account does not have a password.
+Sets the password for the `postgres` user to your specified value.
+
+Default value: `undef`, meaning the superuser account in the Postgres database is a user called `postgres` and this account does not have a password.
```
Comment (jbondpdx) [inline README.md]: Here, I would move the second half of this sentence up. So:

Sets the password for the `postgres` user to your specified value. By default, this setting uses the superuser account in the Postgres database, with a user called `postgres` and no password.

Default value: `undef`.

File: README.md [hunk 3]
```diff
@@ -806,7 +996,9 @@ postgresql::server::config_entry { 'check_function_bodies':
 
 ##### `ensure`
 
-Removes an entry if set to 'absent'. Valid values: 'present', 'absent'.
+Removes an entry if set to 'absent'. 
+
+Valid values: 'present', 'absent'.
```
Comment (jbondpdx) [inline README.md]: Default: 'present'.

(I assume!)

File: README.md [hunk 4]
```diff
@@ -870,55 +1080,79 @@ Creates a database with no users and no permissions.
...
+Valid options: 'ALL', 'CREATE', 'CONNECT', 'TEMPORARY', 'TEMP'.
 
 ##### `psql_db`
 
-Defines the database to execute the grant against. **This should not ordinarily be changed from the default**, which is `postgres`.
+Defines the database to execute the grant against. 
+
+Default value: 'postgres'.
+
+**This should not ordinarily be changed from the default**
```
Comment (jbondpdx) [inline README.md]: I would move this warning sentence up above the Default value, rather than after it.

File: README.md [hunk 5]
```diff
@@ -958,27 +1196,39 @@ Specifies the database to which you are granting access.
...
+Specifies the privilege to grant.
+
+Valid options: 'ALL', 'ALL PRIVILEGES' or 'object_type' dependent string.
 
 ##### `psql_db`
 
-Specifies the database to execute the grant against. _This should not ordinarily be changed from the default_, which is `postgres`.
+Specifies the database to execute the grant against.
+
+_This should not ordinarily be changed from the default_,
```
Comment (jbondpdx) [inline README.md]: Please put this in **bold** instead of _italics_ for consistency with the previous warning like this.

**Comment by pmcmaw** [comment]
@jbondpdx I have now addressed your review comments.

**Comment by jbondpdx** [comment]
Thank you so much again @pmcmaw !

**Pull request merged**